src/handler.cr

```diff
@@ -112,5 +112,33 @@ module Wsman
       @log.info("  Deploying awslogs config #{@awslogs.site_config_path(site_name)}...")
       @awslogs.deploy_site_config(site_name, site.render_awslogs)
     end
+
+    def cleanup(site_name)
+      @log.info("Cleaning up #{site_name}.")
+      @systemd.site_disable_now(site_name)
+      db_config = @config.get_db_config(site_name).first
+      @mysql.drop_db(db_config.dbname, db_config.username)
```

Comment:
In case if a site has more than one database, this code will only delete the first one. Please iterate on the results of `@config.get_db_config` to drop all of them from mysql.
Also please refactor `drop_db` and `drop_user` into separate methods.